docs/spec/v2beta1/helmreleases.md

```diff
@@ -634,6 +634,10 @@ The timeout for any individual Kubernetes operation (like Jobs for hooks) during
 of Helm actions can be configured via `spec.timeout` and can be overridden per action
 via `spec.<action>.timeout`.
 
+> :warning: **If you are using health checks for HelmRelease**: Helm charts containing a single replica will not behave as expected.
```

Comment:
Within the context of the helm-controller, this is not about health checks but about the fact that the wait behaviour does not work as expected (as it does not wait on Helm charts with a single replica).